src/Emulator/Peripherals/Peripherals/Wireless/EFR32xG24_Radio.cs

```diff
@@ -0,0 +1,9178 @@
+                    .WithFlag(28, out RAC_sequencerActive, FieldMode.Read, name: "SEQACTIVE")
+                    .WithTaggedFlag("DEMODENS", 29)
+                    .WithFlag(30, FieldMode.Read, valueProviderCallback: _ => RAC_TxEnable, name: "TXENS")
+                    .WithFlag(31, FieldMode.Read, valueProviderCallback: _ => RAC_RxEnable, name: "RXENS")
+                },
+                {(long)RadioControllerRegisters.Status1, new DoubleWordRegister(this)
+                    .WithValueField(0, 8, FieldMode.Read, valueProviderCallback: _ => RAC_TxEnableMask, name: "TXMASK")
+                    .WithReservedBits(8, 24)
+                }, 
+                {(long)RadioControllerRegisters.Status2, new DoubleWordRegister(this)
+                    .WithEnumField<DoubleWordRegister, RAC_RadioState>(0, 4, FieldMode.Read, valueProviderCallback: _ => RAC_previous1RadioState, name: "PREVSTATE1")
+                    .WithEnumField<DoubleWordRegister, RAC_RadioState>(4, 4, FieldMode.Read, valueProviderCallback: _ => RAC_previous2RadioState, name: "PREVSTATE2")
+                    .WithEnumField<DoubleWordRegister, RAC_RadioState>(8, 4, FieldMode.Read, valueProviderCallback: _ => RAC_previous3RadioState, name: "PREVSTATE3")
+                    .WithEnumField<DoubleWordRegister, RAC_RadioState>(12, 4, FieldMode.Read, valueProviderCallback: _ => RAC_currentRadioState, name: "CURRSTATE")
+                    .WithReservedBits(16, 16)
+                }, 
+                {(long)RadioControllerRegisters.Control, new DoubleWordRegister(this)
+                    .WithFlag(0, out RAC_forceDisable, writeCallback: (_, value) => { if (value) {RAC_UpdateRadioStateMachine();} }, name: "FORCEDISABLE")
+                    .WithTaggedFlag("PRSTXEN", 1)
+                    // Controls automatic 
+                    .WithFlag(2, out RAC_txAfterRx, name: "TXAFTERRX")
+                    .WithTaggedFlag("PRSMODE", 3)
+                    .WithReservedBits(4, 1)
+                    .WithTaggedFlag("PRSCLR", 5)
+                    .WithTaggedFlag("TXPOSTPONE", 6)
+                    .WithTaggedFlag("ACTIVEPOL", 7)
+                    .WithTaggedFlag("PAENPOL", 8)
+                    .WithTaggedFlag("LNAENPOL", 9)
+                    .WithTaggedFlag("PRSRXDIS", 10)
+                    .WithReservedBits(11, 5)
+                    .WithFlag(16, out RAC_prsForceTx, name: "PRSFORCETX")
+                    .WithReservedBits(17, 7)
+                    .WithTaggedFlag("SEQRESET", 24)
+                    .WithFlag(25, out RAC_exitShutdownDisable, FieldMode.Read, name: "EXITSHUTDOWNDIS")
+                    .WithFlag(26, FieldMode.Set, writeCallback: (_, value) =>
+                        {
+                            if(value && sequencer.IsHalted)
+                            {                                
+                                sequencer.SP = machine.SystemBus.ReadDoubleWord(SequencerMemoryBaseAddress);
```

Comment:
It may be necessary to explicitly initialize Vector Table Offset, because otherwise the communication breaks after pause/start. During resuming after pause,  [InitPCAndSP()](https://github.com/SiliconLabs/renode-infrastructure/blob/858570a948b84e8f21dab7d0d7ec97dbf2f18a72/src/Emulator/Cores/Arm-M/CortexM.cs#L295) is called in `OnResume()` and [VectorTableOffset](https://github.com/SiliconLabs/renode-infrastructure/blob/858570a948b84e8f21dab7d0d7ec97dbf2f18a72/src/Emulator/Cores/Arm-M/CortexM.cs#L99) for sequencer should be manually initialized prior to this to not be overridden by the automatically deduced address of the first non null section.
```suggestion
                                sequencer.VectorTableOffset = SequencerMemoryBaseAddress;
                                sequencer.SP = machine.SystemBus.ReadDoubleWord(SequencerMemoryBaseAddress);
```

Reply:
@mars-low thank you for the suggestion. I will give this a try